src/lib/translations/nn-NO/program.json

```diff
@@ -1,6 +1,6 @@
 {
   "title": "Årets program",
-  "description": "Vi gelder oss til ein dag fylt med aktivitetar!",
+  "description": "Vi gelder oss til ein dag fylt med aktivitetar! Informasjon kjem når programmet er klart.",
```

Comment:
TYPO: skal vel være at "vi _gleder_ oss"
Her ville eg brukt "å _gle_ seg", altså "vi _gler_ oss", sjølv om begge former er tillatne

Reply:
Her er det brukt "vi". Om du ønskjer å bruke "me" i staden for "vi", så kan du bytte det ut her.